crates/modelardb_server/src/remote.rs

```diff
@@ -640,7 +640,8 @@ impl FlightService for FlightServiceHandler {
     /// configuration is returned in a [`Configuration`](protocol::Configuration) protobuf message.
     /// * `UpdateConfiguration`: Update a single setting in the configuration. The setting to update
     /// and the new value are provided in the [`UpdateConfiguration`](protocol::UpdateConfiguration)
-    /// protobuf message in the action body.
+    /// protobuf message in the action body. The setting is updated in the live server configuration
+    /// and the change is persisted in the `modelardb.toml` configuration file.
```

Comment:
Update to `modelardbd.toml` if changed above. Alternatively, refer to `CONFIGURATION_FILE_NAME`, but it may be excessive to make it public just for referring to it in the documentation.